components/mailchimp/sources/new-segment-tag-subscriber/new-segment-tag-subscriber.js

```diff
@@ -0,0 +1,97 @@
+const common = require("../common/timer-based");
```

Comment:
To include `constants.mjs`
```suggestion
const common = require("../common/timer-based");
const constants = require("../constants.mjs");
// or import constants from "../constants.mjs";
```